help/destinations/catalog/advertising/Criteo.md

```diff
@@ -0,0 +1,143 @@
+# CRITEO DESTINATION
+
+## Overview
+
+_Criteo powers trusted and impactful advertising to bring richer experiences to every consumer across the open internet. With the world's largest commerce data set and best-in-class AI, Criteo ensures each touchpoint across the shopping journey is personalized to reach customers with the right ad, at the right time._
+
+>[IMPORTANT]
+>
+>This documentation page was created by Criteo. For any inquiries or update requests, please contact Criteo directly [here](https://marketing.criteo.com/).
+
+
+## Prerequisites
+
+- You need to have an administrator user account on [Criteo Management Center](https://marketing.criteo.com)
+- You'll need your Criteo Advertiser ID (Ask your Criteo contact if you don't have this in hand)
+- Criteo only accepts SHA-256 and plain text emails (to be transformed into SHA-256 before sending). Please do not send any PII (Personal Identifiable Information such as individual's names or phone numbers)
+
+![Prerequisites](../../assets/catalog/advertising/criteo/criteo0.png)
+
+
+This is currently a beta product. Get in touch with your Criteo representative if you would like more details about what this means.
```

Comment:
Please move this beta information to the top of the page and wrap it in the following tag:

>[!IMPORTANT]
>
>This is currently a beta product. Get in touch with your Criteo representative if you would like more details about what this means.

Reply:
Done